extensions/wikia/Blogs/Blogs.php

```diff
@@ -112,6 +112,9 @@ function wfBlogsOnBeforeInitialize( &$title, &$article, &$output, User $user, $r
 // Usages of images on blogs on file pages
 $wgHooks['FilePageImageUsageSingleLink'][] = 'BlogsHelper::onFilePageImageUsageSingleLink';
 
+// Checking that user is permitted to delete blog articles
+$wgHooks['BeforeDeletePermissionErrors'][] = 'BlogLockdown::onBeforeDeletePermissionErrors';
```

Comment:
It looks like `BeforeDeletePermissionErrors` will work, but I think the `userCan` might be more appropriate since we're doing a perm check. It gets called right before via [this method](https://github.com/Wikia/app/blob/dev/includes/Article.php#L1388) if you follow the calls down. Thoughts?